CONTRIBUTING.md

```diff
@@ -1,30 +1,29 @@
 # Contributing
 
-Galaxy welcomes new development!
-This document briefly describes how to contribute to the [core
-galaxy project](https://github.com/galaxyproject/galaxy) -
-also checkout our 2013 Galaxy Community
-Conference presentation on the topic
+Galaxy welcomes new development!  This document briefly describes how to
+contribute to the [core galaxy
+project](https://github.com/galaxyproject/galaxy) - also checkout our 2013
+Galaxy Community Conference presentation on the topic
 ([video](https://vimeo.com/channels/581875/73486255),
-[presentation](https://depot.galaxyproject.org/hub/attachments/documents/presentations/gcc2013/BakerContribute.pdf)). For
-information on contributing more broadly to the Galaxy ecosystem and a
+[presentation](https://depot.galaxyproject.org/hub/attachments/documents/presentations/gcc2013/BakerContribute.pdf)).
+For information on contributing more broadly to the Galaxy ecosystem and a
 deeper discussion of some of these points - please see the
-[Develop](https://galaxyproject.org/develop) section of the
-[Galaxy Community Hub](https://galaxyproject.org).
+[Develop](https://galaxyproject.org/develop) section of the [Galaxy Community
```

Comment:
I think we should someday merge the content of https://galaxyproject.org/develop/best-practices/ in `CONTRIBUTING.md` (not asking you to do it, of course!)

Reply:
@nsoranzo Yeah, I was looking at that, too.  This bit about maintaining an updated fork seems like it should be there and not here, among other things.